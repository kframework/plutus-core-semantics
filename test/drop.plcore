(program

(module Foo (imported) (exported () ()) (local () ())
(define b1 (builtin dropByteString 1 #123456))

(define b2 (builtin dropByteString 2 #123456))

(define b3 (builtin dropByteString 3 #123456))
)

)
