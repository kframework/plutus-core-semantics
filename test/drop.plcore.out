<T> <k> . </k> <lenv> .Map </lenv> <genv> ( Foo . b1 ) |-> #23456 ( Foo . b2 ) |-> #3456 ( Foo . b3 ) |-> #456 </genv> <mod> Foo </mod> <sym> .Map </sym> <gas> 9991 </gas> </T>
